pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -155,11 +160,22 @@ private BrokerResponse handleRequest(long requestId, String query,
       compilationStartTimeNs = System.nanoTime();
       switch (sqlNodeAndOptions.getSqlNode().getKind()) {
         case EXPLAIN:
-          String plan = _queryEnvironment.explainQuery(query, sqlNodeAndOptions);
+          queryPlanResult = _queryEnvironment.explainQuery(query, sqlNodeAndOptions);
```

Comment:
For `EXPLAIN` cases, shouldn't we consider deducting the qps quota for that?

Reply:
Not necessarily for now because we don't  send explain plan queries to server.

Reply:
Do we have the plan to support sending EXPLAIN queries to servers? If yes, probably we can add a TODO here in case if it's forgotten?

Reply:
there's no reason to send EXPLAIN PLAN to server since broker has all the information the servers have in order to create the PLAN. the only information on server is whether segment A has inverted index; and segment B doesn't have it (e.g. it was generated with a different table config). and sending per-segment level physical plan info back to user is neither necessary nor practical